releases/data-center/syskit-point-2023-5/point-2023-5-release-note.md

```diff
@@ -0,0 +1,87 @@
+
+* **New File Storage Details** report is available!
+    * Drill from the Site Storage Metrics report to the File Storage Details report by clicking a file name.
+    * Use this report to see all file versions and the total storage used by all versions.
+    * Find how much storage is used by the file versions older than 30 days.
+    * See how much storage is used by the file versions other than the latest 20 versions. 
+    * **Please note that the File Storage Details report is available in the Governance plan only**.
+    * [Read this article to find more information about the File Storage Details report](../../reporting/storage-management/storage-reports.md#file-storage-details).
+
+
+* **The Storage Management feature is now fully available**!
+  * The report section contains the [Site Storage Metrics report](../../reporting/storage-management/storage-reports#site-storage-metrics) and [File Storage Details report](../../reporting/storage-management/storage-reports#site-storage-metrics).
+  * The Site Storage Metrics report has the **Clean Up File Versions action**. 
+    * This means you can select any site, folder, document library, or even **the entire site** and clean up the older file versions.
+  * The Site Storage Metrics report now has a new **File Type tile** added. 
+    * There, you can check how much **storage is occupied by different types of files**: videos, photos, office documents, or others.
+  * [Read this article to learn how to configure Storage Management settings](../../../configuration/configure-storage-management.md).
+  * The Storage Metrics report can have **metadata columns** added by selecting them from the column chooser. 
+  * The File Storage Details report has a **Delete Version action** added when files are selected. 
+  * [Learn more about Storage optimization in the following article](../../../reporting/storage-management/storage-management.md).
+
+* **A new Number of Guest users** condition has been added to the **Rules feature**.
+  * This condition helps define rules that will apply policies to workspaces with or without guest users.
+  * For example, if you want to apply a specific policy to workspaces with Guest users, define a rule with the condition __Number of Guest Users is greater than 0__ and select a policy you want to apply. 
+  * [Learn more about this in the Rules article](../../../governance-and-automation/automated-workflows/policy-automation.md)
+
+
+## Improvements & Bug Fixes
+
+* The **Policy Automation feature** has been **renamed to Rules**, and **Automation within Policies** has been changed to **Task Delegation**. 
+
+* **Improved the descriptions on the policy screen** to enhance the comprehension of each policy.
+
+* **Improved and optimized the Site Metrics report** when showing larger sites.
+
+* **Improved data collection** for storage and site storage quota during autodiscovery.
```

Comment:
ovo staviti na prvo mjesto u Improvements & Bug Fixes
više sam mislio u stilu
* **Autodiscover was optimized.**
      * Optimization results in the Site storage quota and storage used information to be collected faster.
      * As a result, Point now discovers sites with LockState set to NoAccess. Such sites are now visible in Point reports.